### PR TITLE
test(pytest) Import squish as a python module.

### PR DESCRIPTION
**Pull request by StateOf-Vlado**
Import squish as a module.
For automation testing, we want to use PyCharm (https://www.jetbrains.com/pycharm/), but you can use any other IDE or just Terminal
The main goals of these changes are present how to set up an environment to use Squish as a Python module and discuss the framework structure.
Please follow throw steps in the README file.
If you can execute the test, the environment was set up correctly.

If you use PyCharm you are able to execute scripts and tests by clicking one button (Green button on the left side)
image

When you open the test file, you will find the run test button. Try to use debug, to see all methods in squish
Screenshot 2023-07-05 at 15 29 02
image
image

### CODE REVIEW

File: test/ui-pytest/scripts/utils/system_path.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import pathlib
+
+_logger = logging.getLogger(__name__)
+
+
+class SystemPath(pathlib.Path):
+    _accessor = pathlib._normal_accessor  # noqa
+    _flavour = pathlib._windows_flavour if os.name == 'nt' else pathlib._posix_flavour  # noqa
+
+    def rmtree(self, ignore_errors=False):
```
Comment (stefandunca) [inline test/ui-pytest/scripts/utils/system_path.py:12]: Why not use `shutil` instead?

**Review by stefandunca — APPROVED** [review]
Good job. I left one question inline.

**Review by anastasiyaig — APPROVED** [review]
i was able to launch it , but only for x86 binaries.

File: test/ui-pytest/scripts/utils/system_path.py (hunk 1 above)
Comment (StateOf-Vlado) [reply test/ui-pytest/scripts/utils/system_path.py:12]: Fixed, thanks for the comment.

**Comment by StateOf-Vlado** [comment]
I updated README and recorded the setup process for Mac: https://www.notion.so/Setup-Environment-e5d88399027042a0992e85fd9b0e5167?pvs=4
Please if you are interested in automation, try to set up the environment.

**Comment by anastasiyaig** [comment]
to be clear: running pure m1 is not possible:

```
The Squish package does not support the interpreter this script was started with.

The Squish package only has support for x86_64 64-bit Python interpreters.
If you require a package for arm64 64-bit Python please contact qatools.support@qt.io to request such a package.
```


Screenshot 2023-07-07 at 16 07 17

**Comment by StateOf-Vlado** [comment]
> 

Just use installers that presented in README file or follow the video instruction

**Pull request merged**